### PR TITLE
Add adapter function for prepare schema type name

### LINKED ISSUE #102: Do not uppercase root type name
Hasura root type name like 'query_root'
In [line](https://github.com/vuex-orm/plugin-graphql/blob/master/src/graphql/schema.ts#L64) it will uppercase: 'Query_root'
And next I get error: Couldn't find Type of name Query_root in the GraphQL Schema.

### PR DESCRIPTION
**Pull request by ViktorPegy**
Fixes #102

### CODE REVIEW

File: src/graphql/schema.ts [hunk 1]
```diff
@@ -61,7 +64,7 @@ export default class Schema {
   }
 
   public getType(name: string, allowNull: boolean = false): GraphQLType | null {
-    name = upcaseFirstLetter(name);
+    name = this.prepareTypeFunc(name);
```
Comment (phortx) [inline src/graphql/schema.ts]: I think we can call `Context.getInstance().adapter.prepareSchemaTypeName` here and omit the `prepareTypeFunc` param.

**Comment by phortx** [comment]
Hi Viktor,

thanks for your contribution, looks very good :)

I left just a small change request.

Could you also pleas rebase on current master and rebuild the plugin to resolve the conflicts? Thank you very much :)

File: src/graphql/schema.ts (hunk 1 above)
Comment (ViktorPegy) [reply src/graphql/schema.ts]: Fixed :)

**Comment by ViktorPegy** [comment]
Now, wait for release this :)

**Comment by ViktorPegy** [comment]
Hey, can u merge? I wait this fix for dev my project :(

**Comment by ViktorPegy** [comment]
@phortx Hey, can u merge? I wait this fix for dev my project :(

**Comment by phortx** [comment]
Wow, I just totally forgot about this, sorry mate!

**Comment by phortx** [comment]
There are two conflicting files. If you would resolve them, I will merge immediately :)

**Comment by ViktorPegy** [comment]
@phortx, fixed

**Comment by phortx** [comment]
Merged 😎

**Comment by phortx** [comment]
And released: https://github.com/vuex-orm/plugin-graphql/releases/tag/v1.0.0-rc.35

**Comment by ViktorPegy** [comment]
@phortx Very good :)

**Pull request merged**